pkgs/development/compilers/llvm/common/bolt/setup-hook.sh.in

```diff
@@ -0,0 +1,16 @@
+# shellcheck shell=bash
+
+function boltFixupPhase {
```

Comment:
We usually use `boltFixupPhase()` syntax

Reply:
Fixed